Chrysalis-Papilio/Soen-490-Papilio-Android
# PAPILIO-26: Turn off entry to Activity

# General info

**Issue number**: PAPILIO-26

**Task description**: 

 - Allow users to turn off/on entry to their created activity
 - If the activity is closed, then no other user should be able to join this activity

# Testing
To test this feature you have to:

1. Log in to the app
2. Go to the upcoming activities page and tap on your own activity 
3. Tap on the "Turn Off Entry" button and come back to this page again. If you see "Turn On Entry" now, you are good.
4. Now go to Activity Feed and look for the "Montreal HACKnation" activity. 
5. Tap on this activity. If you do not see the join button, you are good. 
6. If steps 3 and 6 are verified successfully, then this story is complete.
